behavior/MouseTracking/FindTrials/find_start_end_of_trials.m

```diff
@@ -0,0 +1,105 @@
+function [ trial_frames, tracking_error_frames] = find_start_end_of_trials( centroids_mat )
+% Using position data from get_mouse_XY_pos (using behavior vid), find
+% frames where trial ends and next trial starts (based on fact that mouse
+% will jump from e.g., south arm to east arm)
```

Comment:
What happens for trials that go like: east-->south, south-->west, west-->north (e.g. on ego-left trials with probe trial in the middle)? 

Is the output of this function intended to be a "suggestion" for the trial boundaries -- i.e. that the user should manually inspect the output? It's fine if so, but it should be explicitly stated.